Doc/library/ast.rst

```diff
@@ -146,6 +146,95 @@ Node classes
     Snakes <https://greentreesnakes.readthedocs.io/en/latest/>`__ project and
     all its contributors.
 
+
+.. _ast-root-nodes:
+
+Root nodes
+^^^^^^^^^^
+
+.. class:: Module(body, type_ignores)
+
+   A Python module.
+
+   *body* is a :class:`list` of the module's :ref:`ast-statements`.
+   *type_ignores* is a :class:`list` of the module's type ignore comments,
+   see :func:`ast.parse` for more details.
+
+   .. doctest::
+
+        >>> print(ast.dump(ast.parse('x = 1'), indent=4))
+        Module(
+            body=[
+                Assign(
+                    targets=[
+                        Name(id='x', ctx=Store())],
+                    value=Constant(value=1))],
+            type_ignores=[])
+
+
+.. class:: Expression(body)
+
+   A single Python expression.
+
+   *body* is a single :ref:`ast-expressions` node.
+
+   .. doctest::
+
+        >>> print(ast.dump(ast.parse('123', mode='eval'), indent=4))
+        Expression(
+            body=Constant(value=123))
+
+
+.. class:: Interactive(body)
+
+   A single Python interactive statement, like in :ref:`tut-interac`.
+
+   *body* is a :class:`list` of :ref:`ast-statements` nodes.
```

Comment:
```suggestion
   *body* is a :class:`list` of :ref:`statement nodes <ast-statements>`.
```

* Fix grammar
* Clarify that we're referring to a list of note types rather than a literal node class `Statement`